### PR TITLE
change the tooltip for private links to be 18F-specific

### PR DESCRIPTION
**Pull request by afeld**
Uses the new feature:

https://github.com/18F/private-eye#reconfiguring-the-default-message

/cc @rogeruiz @abisker @annahsebok

### CODE REVIEW

**Comment by annahsebok** [comment]
:+1:

File: assets/js/18f-guide.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ $(function() {
 
   // https://github.com/18F/private-eye#usage
   PrivateEye({
+    defaultMessage: "This link is private to 18F.",
```
Comment (afeld) [inline assets/js/18f-guide.js]: This is the important part.

Comment (afeld) [inline assets/js/18f-guide.js]: tooltip

**Comment by annahsebok** [comment]
Thanks for pointing out the specifics @afeld. MERGE MERGE MERGE!

**Comment by wslack** [comment]
How do we get this to propagate?

**Comment by afeld** [comment]
https://github.com/18F/guides-template/pull/81 :point_left:

File: lib/guides_style_18f/includes/scripts.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <script src="{% guides_style_18f_asset_root %}/assets/js/vendor/jquery-1.11.2.min.js"></script>
 <script src="{% guides_style_18f_asset_root %}/assets/js/accordion.js"></script>
-<script src="{% guides_style_18f_asset_root %}/assets/js/vendor/private-eye-1.0.0.js"></script>
+<script src="{% guides_style_18f_asset_root %}/assets/js/vendor/private-eye-1.1.0.js"></script>
```
Comment (afeld) [inline lib/guides_style_18f/includes/scripts.html]: Ha...this broke existing sites. Once more in favor of https://github.com/18F/pages-server/issues/55.

**Pull request merged**